tests/tests/trigger_integration/instant.rs

```diff
@@ -50,7 +50,7 @@ async fn poa_instant_trigger_is_produces_instantly() {
         )
         .add_fee_input()
         .finalize_as_transaction();
-        let _tx_id = client.submit(&tx).await.unwrap();
+        let _tx_id = client.submit_and_await_commit(&tx).await.unwrap();
```

Comment:
Do you want to sneak this change or it's leftover ?

Reply:
No, on my machine, this test was failing after the `run` function became non blocking